evm/deploy-upgradable.js

```diff
@@ -38,21 +38,22 @@ async function getImplementationArgs(contractName, config, options) {
         }
 
         case 'AxelarDepositService': {
-            const symbol = contractConfig.wrappedSymbol;
+            const args = options.args;
+            const symbol = args && Array.isArray(args) && args[0] ? args[0] : contractConfig.wrappedSymbol;
 
             if (symbol === undefined) {
                 throw new Error(`Missing AxelarDepositService.wrappedSymbol in the chain info.`);
             } else if (symbol === '') {
                 console.log(`${config.name} | AxelarDepositService.wrappedSymbol: wrapped token is disabled`);
             }
 
-            const refundIssuer = contractConfig.refundIssuer;
+            const refundIssuer = args && Array.isArray(args) && args[1] ? args[1] : contractConfig.refundIssuer;
 
             if (!isAddress(refundIssuer)) {
                 throw new Error(`${config.name} | Missing AxelarDepositService.refundIssuer in the chain info.`);
             }
 
-            const gateway = config.AxelarGateway?.address;
+            const gateway = args && Array.isArray(args) && args[2] ? args[2] : config.AxelarGateway?.address;
```

Comment:
duplicated logic, and pretty unreadable. You can use the same approach as in the above comment.